src/v/cluster/types.h

```diff
@@ -1988,7 +1988,7 @@ struct adl<cluster::begin_tx_request> {
         auto ntp = adl<model::ntp>{}.from(in);
         auto pid = adl<model::producer_identity>{}.from(in);
         auto tx_seq = adl<model::tx_seq>{}.from(in);
-        auto timeout = adl<model::timeout_clock::duration>{}.from(in);
```

Comment:
This is just a hygiene fix, right? The underlying serialization shouldn't be affected, given they both refer to ms?

Reply:
sorta. the type of the field is ms which is what automatic reflection will use. a PR earlier today changed serialization to not use automatic reflection, but rather the explicit serialization seen here. that change used the wrong type.